src/main/java/teammates/ui/controller/Action.java

```diff
@@ -216,6 +214,7 @@ private boolean doesUserNeedToLogin(UserType currentUser) {
         return false;
     }
 
+    @SuppressWarnings("PMD.EmptyIfStmt")
```

Comment:
Not sure if we should. The explanation is in the empty if block itself. I could do a `//NOPMD` but it might suppress future warnings (not that this one doesn't).